cluster/gce/gci/configure-helper.sh

```diff
@@ -95,10 +95,10 @@ function main() {
     gke-master-start
   else
     create-node-pki
-    if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
-      create-kubelet-kubeconfig ${KUBERNETES_MASTER_NAME}
-    else
+    if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]] || [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
```

Comment:
the logic flow is the same if not considering typos for the true/false. if considering typos, the previous one seems better.

Reply:
what typos?

Reply:
ENABLE_APISERVER_INSECURE_PORT=True, or ENABLE_APISERVER_INSECURE_PORT=treu,
previously it will leave to secure mode for kubeconfigs. with this change, it is left not controlled

Reply:
'true' seems to be the correct usage. i=I tested it for scaleout as well. we should only depend on _PARTITION vars. when we enable secure, then insecure port should be used only for debug as it is done today.